fluent-react/src/localization.ts

```diff
@@ -73,6 +79,195 @@ export class ReactLocalization {
     return fallback || id;
   }
 
+  getFragment(
+    id: string,
+    args?: {
+      vars?: Record<string, FluentVariable>,
+      elems?: Record<string, ReactElement>,
+    },
+    fallback?: string
+  ): ReactFragment {
+    return this.getElement(createElement(Fragment, null, fallback || id), id, args);
```

Comment:
Hmm, it's technically not doing much, but I think from the point of view of the mental model of the developer, having the this method still makes sense. The use case is essentially "I want `getString` but with tags in there", rather than "I want this element with the contents replaced".

As an example, I'd prefer to use the following in my code:

```js
  <SomeOtherComponent heading={l10n.getFragment("some-heading", { elems: { b: <b/> } })} />
```

over

```js
  // Not even sure if this is possible:
  <SomeOtherComponent heading={l10n.Element(</>, "some-heading", { elems: { b: <b/> } })} />
  // Otherwise maybe you'd need to do:
  <SomeOtherComponent heading={l10n.Element(<></>, "some-heading", { elems: { b: <b/> } })} />
  // Or even this?
  <SomeOtherComponent heading={l10n.Element(React.createElement(React.Fragment), "some-heading", { elems: { b: <b/> } })} />
```

But let me know, I can remove it if you prefer.

Reply:
It should be pretty easy to define your own sugar to use that API, for example:

```jsx
const getL10nFragment = (id, args) => l10n.getElement(createElement(Fragment, null, id), id, args)

<SomeOtherComponent
  heading={getL10nFragment("some-heading", { elems: { b: <b/> } })}
/>
```

The reason why I'd prefer leaving this out of the API is because it's that easy, and because we're defining a public API that we need to continue to maintain. The smaller the API, the less there is to document and maintain.

Reply:
Yeah that's a fine workaround, and indeed still adding this later would be easier than removing it: 6027f435